example/satosa/pyeudiw_backend.yaml

```diff
@@ -19,11 +19,9 @@ config:
       get_response: '/get-response'
 
   qrcode:
-    size: 100
+    size: 250 #px
```

Comment:
we should handle a specific type, like integer here and say, even with a comment, that the value is expected in pixels

while in the js code or html template we use #px

Reply:
if you say "c'mon Giuseppe #px is a comment" -> ok